node/modules/chain.go

```diff
@@ -82,12 +82,12 @@ func ChainStore(lc fx.Lifecycle,
 	basebs dtypes.BaseBlockstore,
 	weight store.WeightFunc,
 	us stmgr.UpgradeSchedule,
-	j journal.Journal) *store.ChainStore {
+	j journal.Journal) (*store.ChainStore, error) {
 
 	chain := store.NewChainStore(cbs, sbs, ds, weight, j)
 
 	if err := chain.Load(helpers.LifecycleCtx(mctx, lc)); err != nil {
-		log.Warnf("loading chain state from disk: %s", err)
+		return nil, xerrors.Errorf("loading chain state from disk: %w", err)
```

Comment:
Either:

1. We have a large gap in our tests.
2. Failing in this case is "just fine".

Reply:
Note: We'll fail _anyways_ when we call `Start` below.